server/neptune/workflows/internal/deploy/notifier/github_test.go

```diff
@@ -19,6 +19,9 @@ import (
 	"go.temporal.io/sdk/workflow"
 )
 
+var prMode = terraform.PR
+var deployMode = terraform.Deploy
```

Comment:
you didn't remove these?

Reply:
oops looks like they were in two places, removed from here